add .devenv to `projectRootsSearchExclude.default` array (#2643)
BetterThanTomorrow/calva: CHANGELOG.md

## [Unreleased]

## [2.0.478] - 2024-09-30

> - [Calva should not suggest projects in a .devenv/ directory when starting a project REPL and connecting (jack-in)](https://github.com/BetterThanTomorrow/calva/issues/2642)

Reviewer: ```suggestion
- [Stop Calva from, by default, including .devenv/ directory when starting a project REPL](https://github.com/BetterThanTomorrow/calva/issues/2642)
```
